src/components/FullImage/FullImage.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+  <div v-if="this.imageData" class="information">
+    <el-row>
+      <cross @click="backToCanvasView()" />
+    </el-row>
+    <el-row>
+      <h1>
+        {{ this.imageData[0].title }}
+      </h1>
+    </el-row>
+    <el-row>
+      <h2>Author</h2>
+    </el-row>
+    <el-row>
+      <h3>
+        {{ this.imageData[0].author }}
+      </h3>
+    </el-row>
+    <el-row>
+      <p>
+        {{ this.imageData[0].medium }}
+      </p>
+    </el-row>
+    <el-row v-if="collection">
+      <p>{{ currentImage + 1 }} / {{ collection.length }}</p>
+    </el-row>
+    <el-row>
+      <div style="display: flex; cursor: pointer" @click="previousImage()">
+        <arrow-left />
+        <p>&nbsp; Previous &nbsp;</p>
+      </div>
+      <p>|</p>
+      <div style="display: flex; cursor: pointer" @click="nextImage()">
+        <p>&nbsp; Next &nbsp;</p>
+        <arrow-right />
+      </div>
+    </el-row>
+    <el-row>
+      <span>{{ this.imageData[0].description }}</span>
+    </el-row>
+  </div>
+  <div class="page">
+    <div id="viewer" class="viewer"></div>
+  </div>
+</template>
+
+<script>
+import OpenSeadragon from "openseadragon";
+import Cross from "./Cross.vue";
+import ArrowLeft from "./ArrowLeft.vue";
+import ArrowRight from "./ArrowRight.vue";
+
+import dataFetching from "../../api/dataFetching";
+
+export default {
+  components: { Cross, ArrowLeft, ArrowRight },
+  beforeRouteEnter(to, from, next) {
+    next((vm) => {
+      vm.imageId = to.params.index;
+    });
+  },
+  data() {
+    return {
+      imageId: undefined,
+      currentImage: undefined,
+      imageData: undefined,
+      collection: undefined,
+      viewer: undefined,
+    };
+  },
+  methods: {
+    previousImage() {
+      if (this.currentImage > 0) {
+        this.currentImage = this.currentImage - 1;
+        this.imageData[0] = this.collection[this.currentImage];
```

Comment:
does not need to be an array, it sound sover complicated to do `this.imageData[0]` all the time, just do `this.imageData = `